packages/backend-test-utils/src/database/types.ts

```diff
@@ -43,40 +43,44 @@ export type Instance = {
   connections: Array<Knex>;
 };
 
+const DOCKER_REGISTRY_PREFIX = process.env.BACKSTAGE_TEST_DOCKER_REGISTRY
+  ? `${process.env.BACKSTAGE_TEST_DOCKER_REGISTRY}/`
+  : '';
+
 export const allDatabases: Record<TestDatabaseId, TestDatabaseProperties> =
   Object.freeze({
     POSTGRES_13: {
       name: 'Postgres 13.x',
       driver: 'pg',
-      dockerImageName: 'postgres:13',
+      dockerImageName: `${DOCKER_REGISTRY_PREFIX}postgres:13`,
```

Comment:
I think it would be a little cleaner here to have a utility function instead, that we could also use in other places too when needed. but something like `getDockerImageForName('postgres:13')` or something?

Reply:
Good idea, will refactor!